instagram-app/src/components/CommentSection/CommentForm.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const CommentForm = props => {
+  return (
+    <form onSubmit={props.submitComment}>
+      <input
+        type="text"
+        value={props.comment}
+        placeholder="Make a Comment!"
+        onChange={props.changeComment}
+      />
```

Comment:
Nice work on this form. Functionality works nicely. Code is clean and easy to read.